lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -223,10 +224,16 @@ def main():
     if "delta_time" in config[clean_method_name]:
         delta_time = config[clean_method_name]["delta_time"]
 
-    subarray_info = SubarrayDescription.from_hdf(args.input_file)
     tel_id = config["allowed_tels"][0] if "allowed_tels" in config else 1
-    optics = subarray_info.tel[tel_id].optics
-    camera_geom = subarray_info.tel[tel_id].camera.geometry
+    try:
+        subarray_info = SubarrayDescription.from_hdf(args.input_file)
+        optics = subarray_info.tel[tel_id].optics
+        camera_geom = subarray_info.tel[tel_id].camera.geometry
+    except OSError:
```

Comment:
Is `OSError` raised when trying to run with a v0.9 file without the try block?

Reply:
Yes, I just used the same as in previous fixes of the same problem (in dl1 to dl2). This is what it yields:
OSError: Unsupported version of subarray table: 1.0